sematic/ui/src/components/GitInfo.tsx

```diff
@@ -37,78 +48,56 @@ function GitInfo(props: {
   }, [text]);
 
   return (
-
-    <Typography color="GrayText" component="span">
+    <Typography
+      color="GrayText"
+      component="span"
+      sx={{ display: "flex", alignItems: "center" }}
+    >
+      {children}&nbsp;
       <Tooltip title={tooltip}>
-        <Box component="span">
-          {children}&nbsp;
-          <Link href={makeGithubLink(remote, path)} target="_blank">
-            <code>{content}</code>
-          </Link>
-          <ButtonBase onClick={copy}>
-            <ContentCopy fontSize="inherit" sx={{ ml: 1 }}/>
-          </ButtonBase>
-        </Box>
+        <Link href={makeGithubLink(remote, path)} target="_blank">
+          {content}
```

Comment:
Please keep the code formatting. Hashes are already code-formatted elsewhere in the UI, and monospaced formatting prevents the "Copied" message from reshaping the component once the copy button is clicked.